### PR TITLE
Feat/interest rate hashtack v1

### PR DESCRIPTION
**Pull request by djeck1432**
1. Added interest rate hashtack v1 new events
2. Added new migrations file for new field `version`
3. Added logic to store `debt_category`
4. ADded code for first running
5. Regacroting Hashtackv0 and hashtackv1

### CODE REVIEW

File: data_handler/handlers/loan_states/hashtack_v1/events.py [hunk 1]
```diff
@@ -390,6 +397,56 @@ def __init__(
...
+            - total_supply: 2
+            - total_debt: 3
+            - timestamp: 4
+        :param event: pd.Series with event data
+        :return: None
+        """
+        token = get_symbol(event["data"][1])
+        # Convert total_supply and total_assets from hex to Decimal
+        total_supply = decimal.Decimal(str(int(event["data"][2], base=16)))
+        total_debt = decimal.Decimal(str(int(event["data"][4], base=16)))
```
Comment (nickgore) [inline data_handler/handlers/loan_states/hashtack_v1/events.py:439]: in docstring it says` total_debt: 3`. Please double check what is correct index to use here.

File: data_handler/handlers/loan_states/abstractions.py [hunk 2]
```diff
@@ -262,3 +268,52 @@ def run(self) -> None:
...
+
+        filtered_loan_entities: dict = {}
+        # remove objects from loan_entities_values if the object has `has_skip` attribute
+        for loan_id, loan_entity in loan_entities.items():
+            has_skip = getattr(loan_entity, "has_skip", None)
+            if not has_skip:
+                filtered_loan_entities[loan_id] = loan_entity
+
+        # if there are no loan entities, return an empty DataFrame
+        if not loan_entities:
```
[the remark this answers is not in the document]
Comment (djeck1432) [reply data_handler/handlers/loan_states/abstractions.py:291]: No, the main point is in `loan_entities` so I need to check `loan_entities` in condition. `filtered_loan_entities` could be sometimes empty sometimes no

[the remark this answers is not in the document]
Comment (djeck1432) [reply data_handler/handlers/loan_states/abstractions.py:291]: p.s. I changed order of this checking to add there more sense

File: data_handler/handlers/loan_states/hashtack_v1/events.py (hunk 1 above)
Comment (djeck1432) [reply data_handler/handlers/loan_states/hashtack_v1/events.py:439]: yeah, good catch, it should be 4, thanks

**Pull request merged**